script.js

```diff
@@ -1,4 +1,210 @@
+var draws = 0;
+var playerWins = 0;
+var computerWins = 0;
+var timesPlayed = 0;
+var previousWinner = '';
+
+var mode = 'waiting for playerName';
+var playerName = '';
+
+// generate a random number between 1 and 3
+var randomNumber = function () {
+  return (Math.floor(Math.random() * 3)) + 1;
+};
+
+// game block
+var playGame = function (playerGuess) {
+  var computerChoice = '';
+  var computerNumber = randomNumber();
+  var message = '';
+
+  if (computerNumber == 1) {
+    computerChoice = 'scissors';
+  } else if (computerNumber == 2) {
+    computerChoice = 'paper';
+  } else {
+    computerChoice = 'stone';
+  }
+
+  // if both player and computer choose the same
+  if (playerGuess == computerChoice) {
+    draws += 1;
+    message = `${playerName}, it's a DRAW! The previous winner ${previousWinner} wins!<br><br>Your choice: ${playerGuess}<br>Computer's choice: ${computerChoice}<br>Player wins: ${playerWins}
+    <br>Computer wins: ${computerWins}<br>Draws: ${draws}<br>You have won ${playerWins}/${timesPlayed} times`;
```

Comment:
Generation of this message can be abstracted to a helper function. @karen-heng should be able to help with this :)